src/services/avatars.ts

```diff
@@ -0,0 +1,166 @@
+import { Service } from "../service";
+
+interface assoc {
+    [key: string]: any;
+}
+
+export class Avatars extends Service {
+
+    /**
+     * Get Browser Icon
+     *
+     * You can use this endpoint to show different browser icons to your users.
+     * The code argument receives the browser code as it appears in your user
+     * /account/sessions endpoint. Use width, height and quality arguments to
+     * change the output settings.
+     *
+     * @param string code
+     * @param number width
+     * @param number height
+     * @param number quality
+     * @throws Exception
+     * @return Promise<string>
+     */
+    async getBrowser(code: string, width: number = 100, height: number = 100, quality: number = 100): Promise<string> {
+        let path = '/avatars/browsers/{code}'.replace(new RegExp('{code}', 'g'), code);
+        
+        return await this.client.call('get', path, {
+                    'content-type': 'application/json',
+               },
+               {
+                'width': width,
+                'height': height,
+                'quality': quality
+            });
+    }
+
+    /**
+     * Get Credit Card Icon
+     *
+     * Need to display your users with your billing method or their payment
+     * methods? The credit card endpoint will return you the icon of the credit
+     * card provider you need. Use width, height and quality arguments to change
+     * the output settings.
+     *
+     * @param string code
+     * @param number width
+     * @param number height
+     * @param number quality
+     * @throws Exception
+     * @return Promise<string>
+     */
+    async getCreditCard(code: string, width: number = 100, height: number = 100, quality: number = 100): Promise<string> {
+        let path = '/avatars/credit-cards/{code}'.replace(new RegExp('{code}', 'g'), code);
```

Comment:
I already commented about the let, so it's for all the let variables you defined.

use `const` when you know that (for example in this func) path will never change. use `let` when you need to re-assign values.

Regarding the RegExp - just use backticks.

for example: ```
```const avatar = 'someUrl';```
 let path=`/avatars/${avatar}`
 ```;